docs/contributing/index.rst

```diff
@@ -3,59 +3,26 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
-coding, graphics design, documentation or sponsorship:
+`Translate <https://hosted.weblate.org/projects/weblate/>`_ Weblate itself,
+and point out errors in source strings while you do. Suggest source string changes, or
+click "Source string location" in the lower right of the regular editor to have
+a go yourself.
 
-* :doc:`issues`
-* :doc:`start`
-* :ref:`translating-weblate`
-* :ref:`contributing-docs`
-* :ref:`funding-weblate`
+Here in the documentation, click :guilabel:`Edit on GitHub` atop the page to do the same thing.
+`Open an issue https://github.com/WeblateOrg/weblate/issues/new?assignees=comradekingu&labels=ux&template=bug_report.md>`_
+if you know something is wrong or have an issue with the documentation, but not how to fix it.
+Remove the template stuff; a link and description will do.
 
-.. _translating-weblate:
+Funding
+-------
 
-Translating Weblate
--------------------
-
-Weblate is being `translated <https://hosted.weblate.org/>`_ using Weblate itself, feel
-free to take part in the effort of making Weblate available in as many human languages
-as possible.
-
-.. _contributing-docs:
-
-Contribute to Weblate documentation
------------------------------------
-
-You are welcome to improve any documentation page.
```

Comment:
Instead of all the "this happened once". Let's not make reading about it a hurdle to overcome for every new contributor.

Reply:
Obstacles during merging docs' PRs are often. Also, many clients asked me for this so we have the rules that can be improved, not removed.

I am sorry, but I cannot let you do the documentation merges. You changed the meaning of the sentences many times. PRs also contained changes that introduced descriptions not according to the reality of the app. Everybody can make mistakes so we have multi-level reviews :)